pom.xml

```diff
@@ -50,22 +57,31 @@
     </repository>
   </repositories>
 
-  <distributionManagement>
-    <repository>
-      <id>folio-nexus</id>
-      <name>FOLIO Release Repository</name>
-      <url>https://repository.folio.org/repository/maven-releases/</url>
-      <uniqueVersion>false</uniqueVersion>
-      <layout>default</layout>
-    </repository>
-    <snapshotRepository>
-      <id>folio-nexus</id>
-      <name>FOLIO Snapshot Repository</name>
-      <uniqueVersion>true</uniqueVersion>
-      <url>https://repository.folio.org/repository/maven-snapshots/</url>
-      <layout>default</layout>
-    </snapshotRepository>
-  </distributionManagement>
```

Comment:
Why have these been moved?

Reply:
I just moved this section at the end of project file, after dependencies part.

Reply:
I understand that is what you did. Why did you do it?

Reply:
Just when checking other modules I saw this section as usual specified at the end of pom config, so I just moved it to have same structure. But I agree it's better do make bare-minimum changes in files to simplify code reading. I'll compare it with mainline branch and revert this part of changes back. Thank you for pointing this.

Reply:
> I agree it's better do make bare-minimum changes in files to simplify code reading.

I wasn't expressing an opinion, only trying to understand why. If you'd like to split this change into another pull request that would be cool. I could approve that quickly.